linux_os/guide/system/network/network-wireless/wireless_software/wireless_disable_interfaces/rule.yml

```
@@ -31,7 +31,7 @@ identifiers:
 references:
     stigid@rhel6: "000293"
     stigid@rhel7: "041010"
-    cis: 4.3.1
+    cis@rhel8: 3.5
```

```suggestion
    cis@rhel8: "3.5"
```
The build system interprets as a float number in this case. It needs to be double quoted

---

Ack & fixed